lib/colorado_lottery.rb

```diff
@@ -0,0 +1,19 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    return true if contestant.age < 18 && contestant.game_interests.include?(game.name)
+    false
+  end
+
+  def can_register?
+    return true if (interested_and_18?(contestant, game) == true) && (contestant.out_of_state? && game.national_drawing?) == true
```

Comment:
No need for the `== true` as the `interested_and_18?` method will return a boolean. We also needed `(!contestant.out_of_state? || game.national_drawing?)` because  the contestant can register if either of those statements return a true, so we only need one to be true, which makes it an or statement.